.gitignore

```diff
@@ -1 +1,2 @@
 **/__pycache__
+/dist
```

Comment:
Maybe fill this file with the default github's python .gitignore: https://github.com/github/gitignore/blob/main/Python.gitignore

Reply:
Created issue #77